### PR TITLE
upgrade django to 4.2

### PR DESCRIPTION
**Pull request by aminebz949904**
upgrade django to 4.2

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+django==4.2.8
 django-environ
 django-jquery
 psycopg2-binary
 python-dotenv
 requests
 raven
 sentry_sdk
-django-staff-sso-client
+django-staff-sso-client==4.2.1
```
Comment (adamwozencroft) [inline requirements.in:11]: @aminebz949904 does this `django-staff-sso-client` need a version?

Comment (aminebz949904) [reply requirements.in:11]: Not sure why without the version number the pip-compile throw a dependency exception..and by adding it it works

Comment (adamwozencroft) [reply requirements.in:11]: Upgrade to Django=4.2.8 forces `django-staff-sso-client` to use as newer release, as old Django is not supported. More details here: 
https://github.com/uktrade/django-staff-sso-client

**Review by adamwozencroft — APPROVED** [review]
Approved, thanks Amine!

**Pull request merged**